src/Microsoft.ML.StandardLearners/Microsoft.ML.StandardLearners.csproj

```diff
@@ -10,6 +10,7 @@
     <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
     <ProjectReference Include="..\Microsoft.ML.CpuMath\Microsoft.ML.CpuMath.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.FastTree\Microsoft.ML.FastTree.csproj" />
```

Comment:
><ProjectReference Include="..\Microsoft.ML.FastTree\Microsoft.ML.FastTree.csproj" /> [](start = 4, length = 84)

Is this necessary?

Reply:
it's actually PKPD referencing FastTree in the documentation through a <see> that adds this project reference. 

---
In reply to: [202456059](https://github.com/dotnet/machinelearning/pull/529#discussion_r202456059) [](ancestors = 202456059)

Reply:
Yes I'd prefer to not have things like this result in dependencies. :)

---
In reply to: [202741801](https://github.com/dotnet/machinelearning/pull/529#discussion_r202741801) [](ancestors = 202741801,202456059)